### ISSUE TITLE
case-insensitive matching ? 

### ISSUE THREAD

**Issue by mogando668** [JorySmith/DrAwkwardPalindrome#1]
i just tested my own name reversed, but intentionally up-cased one single letter on right hand side,

jasonknoSaj

and it came back saying it's a match. is that intentional ?

**Comment by JorySmith** [comment]
Yes, this program is not looking for a perfect case match, just letters that are arranged the same way backward and forward.

**Comment by mogando668** [comment]
oh one more issue too :  i tested it with a valid UTF8 input of

`123덈321
`
corresponding to bytes :
```
0000000        3946000945       842238093              49                
           1   2   3  덈  **  **   3   2   1                            
          061 062 063 353 215 210 063 062 061                            
           1   2   3   ?  8d  88   3   2   1                            

           49  50  51 235 141 136  51  50  49                            
           31  32  33  eb  8d  88  33  32  31                            
0000011
```
In emacs meta notation, it would be 

`123M-kM-^MM-^H321
`

it's a palindrome on UTF8 level, but not a palindrome on byte/posix-c level. 

the app came back clearing all input, as if re-started from a clean slate, without confirming whether it's match or not.

**Comment by mogando668** [comment]
ps : third comment :

you only need to reverse the 2nd-half of the string, then either use regex or index( ) type of function to check whether the matching point is exactly at starting position or not.
